controlnext/models.py

```diff
@@ -187,13 +186,14 @@ def __getattr__(self, item):
 
 class UserProfile(models.Model):
     user = models.ForeignKey(User, verbose_name=_("user"))
-    grower = models.ForeignKey(GrowerInfo, verbose_name=_("grower"))
+    grower = models.ManyToManyField(GrowerInfo, verbose_name=_("grower"))
 
     def __unicode__(self):
         identifier = self.user if self.user else self.id
-        return "{} ({})".format(self.grower, identifier)
+        return "{} ({})".format(identifier, ', '.join([str(x) for x in
```

Comment:
Alarmbelletje: je moet unicode teruggeven in `__unicode__`. Dus `str(x)` is een tikkie gevaarlijk.

Ik zou sowieso voor alle modellen even een piepkleine test toevoegen voor deze methode, want dat is de plek waar ik zelf het meest een fout maak. En die zie je pas als je een foutmelding wilt printen (wat vervolgens fout gaat omdat de `__unicode__()` een foutje bevat...)....